src/databricks/sqlalchemy/dialect/__init__.py

```diff
@@ -47,7 +47,7 @@ class DatabricksTimestamp(types.TypeDecorator):
     impl = types.TIMESTAMP
 
     def process_result_value(self, value, dialect):
-        return processors.str_to_datetime(value)
+        return value
```

Comment:
what about date columns?

Reply:
Fixed in c520103f309275f28850ab6c5b3b8d4cfa3ff245